distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestSecurity.java

```diff
@@ -230,6 +230,28 @@ public void testBasicRestAccess() throws Exception {
     }
 
     @Test
+    public void testSamlFederatedAuth() throws Exception {
+        String recordId = TestCatalog.ingest(Library.getSimpleGeoJson(), "application/json");
+        configureRestForBasic();
+
+        // Creating a new OpenSearch source with no username/password.
+        // When an OpenSearch source attempts to authenticate without a username/password it will
+        // use the subject in the request to create a SAML authentication token
+        OpenSearchSourceProperties openSearchProperties = new OpenSearchSourceProperties(
+                OPENSEARCH_SOURCE_ID);
+        createManagedService(OpenSearchSourceProperties.FACTORY_PID, openSearchProperties);
+
+        waitForFederatedSource(OPENSEARCH_SOURCE_ID);
+
+        String openSearchQuery = SERVICE_ROOT + "/catalog/query?q=*&src=" + OPENSEARCH_SOURCE_ID;
+        given().auth().basic("admin", "admin").when().get(openSearchQuery).then().log().all()
+                .assertThat().statusCode(equalTo(200)).assertThat().body(containsString("myTitle"));
+
+        configureRestForAnonymous();
```

Comment:
I don't think that should be addressed with a `try/finally`. I think it's a pre-condition that should be enforced by the `@BeforeTest` or similar. Ideally all itest cleanup happens in the same place and guarantees a consistent state for any subsequent itests. This is also a step towards being able to run all the itests sequentially in the PaxExam instance, which would help reduce the time that running itests takes.